rero_ils/modules/notifications/api.py

```diff
@@ -131,7 +131,7 @@ def replace_pids_and_refs(self):
                 data['loan']['author'] = author
             end_date = data.get('loan').get('end_date')
             if end_date:
-                end_date = ciso8601.parse_datetime_as_naive(end_date)
+                end_date = ciso8601.parse_datetime(end_date)
```

Comment:
can you use `date_string_to_utc` instead?

Reply:
done